pyproject.toml

```diff
@@ -1,2 +1,17 @@
 [build-system]
-requires = ["setuptools", "wheel", "Cython"]
+requires = ["setuptools", "wheel", "Cython","numpy"]
+[tool.cibuildwheel]
+before-all = "bash package/scripts/install_prereqs_for_wheels.sh"
+before-build = [
+"pip install -r requirements_dev.txt",
+]
+test-command = "python {project}/test/test.py"
+[tool.cibuildwheel.linux]
+skip = "pp* *-manylinux_i686 *_ppc64le *_s390x *musll*"
```

Comment:
Can you add some comments here about what platforms are actually getting built? Probably add pointers to manylinux nomenclature.